pallet/nft/src/lib.rs

```diff
@@ -49,6 +52,8 @@ use sp_std::prelude::*;
 mod benchmarking;
 #[cfg(test)]
 pub mod mock;
+#[cfg(feature = "std")]
```

Comment:
only import for tests instead?
```suggestion
#[cfg(test)]
```

Reply:
I think we talked about this previously but other pallets can't access the test_utils if it is behind the test flag, hence why we decided to use std